ansible_ai_connect/ai/api/model_client/wca_client.py

```diff
@@ -592,15 +612,18 @@ def post_request():
         playbook = response["playbook"]
         outline = response["outline"]
 
-        if ansible_lint_caller := apps.get_app_config("ai").get_ansible_lint_caller():
+        from ansible_ai_connect.ai.apps import AiConfig
```

Comment:
I'm more intrigued by this change than anything... why the need?

Reply:
It's because the get_app_config returns different type of config depending on the App, and Pyright was not happy because of an undefined method. I'm not totally sure Why the code change triggered this behavior.